inst/components/sidebar.scss

```diff
@@ -13,8 +13,9 @@ $bslib-sidebar-border: var(--bs-card-border-width, #{$card-border-width}) solid
   --bslib-collapse-toggle-right-transform: -90deg;
 
   display: grid !important;
-  // Should always be defined as an inline style on the layout_sidebar() container
-  grid-template-columns: var(--bslib-sidebar-columns);
+  grid-template-columns:
+    Min(calc(100% - #{$bslib-sidebar-icon-size}), var(--bslib-sidebar-width, 250px))
+    minmax(0, 1fr);
```

Comment:
Let's consider putting each column definition into a CSS variable (defined above), for example:


```css
---bslib-sidebar-width-actual: Min(calc(100% - #{$bslib-sidebar-icon-size}), var(--bslib-sidebar-width, 250px))
```

then when we need it:

```css
grid-template-columns: var(---bslib-sidebar-width-actual) var(---bslib-main-column)
```

Reply:
Fixed in 86afeec9710ea27459c6129f2fdef15188d5ef11

I went with two sass variables, `$bslib-sidebar-column-sidebar` and `$bslib-sidebar-column-main`. Overall a definite improvement! I stuck with SASS variables because we don't need to expose these client-side. The sidebar column width carries the `--bslib-sidebar-width` variable, so that can be used client side to adjust sidebar width (for the adventurous slash if needed).